### ISSUE TITLE
Incorrect blend mode used

### ISSUE THREAD

**Issue by Sheap** [StereoKit/StereoKit#531]
I'm setting blendPreference to DisplayBlend.AnyTransparent, but when the app runs, it's using opaque, and my view is covered in the clear color, making it hard to see objects.

My headset is a project northstar DeckX, so I suspect that the issue is that the device type is being improperly communicated somewhere. How is it determined? Can I force the blend mode on the renderer somehow?

**Comment by maluoi** [comment]
What OpenXR runtime are you using? I know I've seen North Star headsets in the past that have reported themselves as opaque, which is almost certainly what's happening here. I believe the Monado runtime may account for this in some form or another now, but I'm uncertain about what SteamVR does! You may need to take it up with the runtime maintainers.

You can get the same behavior from StereoKit by switching a few settings around! I have to do this for Quest passthrough, which you can [see over here](https://github.com/StereoKit/StereoKit/blob/master/Examples/StereoKitTest/Tools/PassthroughFBExt.cs#L83).

Basically just these two items:
```csharp
Renderer.ClearColor = Color.BlackTransparent;
Renderer.EnableSky  = false;
```

**Comment by Sheap** [comment]
Sorry for the late response - I hadn't had a chance to test it. This worked great, thanks!
